Automattic/wp-calypso
# Get Apps: Migrate Image to Webpack

#### Changes proposed in this Pull Request

This migrates the one image in Me section (I think) to Webpack. For details on the reasoning behind this, see #36850.

#### Testing instructions

There should only be one use of this image: https://github.com/Automattic/wp-calypso/search?q=get-apps-banner&unscoped_q=get-apps-banner

You can find it by visiting `/me/get-apps`. Verify that the image is still there and no different. 

<img width="698" alt="Screenshot 2019-11-16 at 15 08 21" src="https://user-images.githubusercontent.com/43215253/68995223-2011d900-0883-11ea-872d-07bee51eb0ae.png">
